Usage.md

```diff
@@ -16,21 +16,27 @@ The app will start with subscriptions defined in `resources/subscription-example
 
 When the app has started, an admin-gui will be available at http://localhost:8012/anshar/stats 
 
-# Using docker
+# Using the container image
 
-Build the image :
```

Comment:
The command you build will try to push the image to GHCR. Since this is not actually using docker it's a bit complicated to build a local image with jib: https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#build-your-image

You'd have to build into a tar and then import it into your local docker installation.

Reply:
Would you find this useful information?

Reply:
No, importing a tar into Docker/podman seems to be less common, and people who know their tools that well can lookup how the plugin's docs.
What do you think?

Reply:
I don't think it's useful information and can be left out.